### PR TITLE
bugfix/redesign/UBS.Main-page.extra-pop-up

### PR DESCRIPTION
**Pull request by AlexW8719**

### CODE REVIEW

**Review by KKatiaa — APPROVED** [review]

File: src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss [hunk 1]
```diff
@@ -720,11 +723,14 @@ section .main-container {
...
       .main-container {
         height: 100%;
-        margin: 0;
         padding: 0 150px 117px 150px;
 
         .box {
           margin-top: 92px;
+          margin-left: auto;
+          margin-right: auto;
+          // width: 100%;
```
Comment (katia9261) [inline src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss]: Don't forget to remove comment)

File: src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss [hunk 2]
```diff
@@ -1758,9 +1787,9 @@ section .main-container {
       height: 100%;
 
       .box-how-works {
-        margin-top: 60px;
-        margin-bottom: 60px;
-        padding: 0 16px 0 16px;
+        // margin-top: 60px;
```
Comment (katia9261) [inline src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss:1789]: Here also comments

**Review by katia9261 — APPROVED** [review]

File: src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss (hunk 2 above)
Comment (AlexW8719) [reply src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss:1789]: I'll fixed it in next push soon

File: src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss [hunk 3]
```diff
@@ -634,7 +634,7 @@ section .main-container {
     margin: 0 auto;
     padding: 64px 0 0 0;
     font-size: 16px;
-    max-width: 1440px;
+    max-width: 1920px;
```
Comment (atsa21) [inline src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss:637]: Screen of user can be more that 1920 - max-width: 100%

File: src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss [hunk 4]
```diff
@@ -1758,9 +1786,9 @@ section .main-container {
       height: 100%;
 
       .box-how-works {
-        margin-top: 60px;
-        margin-bottom: 60px;
-        padding: 0 16px 0 16px;
+        // margin-top: 60px;
+        // margin-bottom: 60px;
```
Comment (atsa21) [inline src/app/ubs/ubs/components/ubs-main-page/ubs-main-page.component.scss:1790]: Still see comment, need to remove it

**Review by atsa21 — APPROVED** [review]

**Review by NadiiaStoiko — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**